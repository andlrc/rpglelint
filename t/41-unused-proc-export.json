[
   {
      "column":10,
      "file":"t/41-unused-proc-export.rpgle",
      "line":9,
      "message":"'c' defined but not used",
      "severity":"warning"
   }
]
