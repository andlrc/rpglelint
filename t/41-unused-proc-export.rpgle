// rpglelint: -Wunused-procedure
dcl-proc a export;
  b();
end-proc;

dcl-proc b;
end-proc;

dcl-proc c;
end-proc;
