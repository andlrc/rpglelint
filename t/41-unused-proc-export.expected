t/41-unused-proc-export.rpgle:9:10: warning: 'c' defined but not used [-Wunused-procedure]
 dcl-proc c;
          ^
